tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Linker.scala

```diff
@@ -243,6 +270,43 @@ final class Linker(semantics: Semantics, outputMode: OutputMode,
         version)
   }
 
+  private def synthesizeInheritedConstructor(
+      classInfo: Analysis.ClassInfo, methodInfo: Analysis.MethodInfo,
+      getTree: TreeProvider, analysis: Analysis)(
+      implicit pos: Position): MethodDef = {
+    val encodedName = methodInfo.encodedName
```

Comment:
Consider replacing this by a constant. This code really only ever makes sense for ctors. So no point to generalize. It's just confusing.